style.css

```diff
@@ -706,3 +699,209 @@ form button {
   padding-top: 40px;
   color: #fff;
 }
+
+.pop-window {
+  position: fixed;
+  background-color: white;
+  border-radius: 5%;
+  top: 2%;
+  bottom: 2%;
+  left: 2%;
+  right: 2%;
+  flex-direction: column;
+  padding-left: 16px;
+  padding-right: 16px;
+  display: flex;
+  padding-bottom: 1%;
+  overflow-y: scroll;
+}
+
+.headline-cross {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  padding-top: 30px;
+}
+
+.headline-cross h2 {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.headline-cross .cross {
+  padding-right: 30px;
+  font-weight: bold;
+  cursor: pointer;
+  align-self: center;
+}
+
+.subtitles {
+  display: flex;
+  column-gap: 12px;
+  margin-bottom: 10px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-weight: 600;
+  font-size: 13px;
+  color: #7a869a;
+  list-style: none;
+}
+
+.mobile-image img {
+  height: auto;
+}
+
+.pop-window ul {
+  display: flex;
+  padding: 0;
+  gap: 5px;
+}
+
+.pop-window .languages li {
+  margin-right: 5px;
+  background-color: rgb(226, 219, 219);
+  padding: 0 6px;
+  border-radius: 20%;
+}
+
+.mobile-paragraph {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-bottom: 6px;
+  margin-top: 6px;
+}
+
+.pop-window .languages {
+  display: flex;
+  column-gap: 8px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.languages ul {
+  padding: 0;
+  border-radius: 20%;
+  margin-top: 0;
+}
+
+@media (min-width: 768px) {
+  .pop-window .languages {
+    display: flex;
+    column-gap: 8px;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    color: #6070ff;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding: 0;
+    margin-right: 10%;
+  }
+
+  .buttons {
+    display: flex;
+    gap: 10px;
+    margin-top: 10px;
+    float: right;
+  }
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided.
- Please ensure that those buttons and labels have the same style, dimensions, and position provided by the Figma template. :eyes: :eyes: 

Your desktop design
![image](https://user-images.githubusercontent.com/98363075/202189988-8c7f0d01-44f7-4d91-8838-1c47e86a1ebf.png)

Figma desktop design
![image](https://user-images.githubusercontent.com/98363075/202190101-90374cec-d6e0-470f-9622-1838f7e69655.png)

Your mobile design
![image](https://user-images.githubusercontent.com/98363075/202190518-be73f42a-4f7c-4444-9f94-92c2e000c571.png)

Figma mobile design
![image](https://user-images.githubusercontent.com/98363075/202190773-940ad862-5666-4ea0-9408-ae839e1ff819.png)